### PR TITLE
Configurable package property

### PR DESCRIPTION
**Pull request by fpirola**
Create package/folders based on a new property "package" defined in default.properties.

This changes get ideas based on [lagom scala g8](https://github.com/lagom/lagom-scala.g8)

### CODE REVIEW

File: src/main/g8/default.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
 description = This template generates a Play Scala project
 name = play-scala-seed
 organization=com.example
+package = $organization$.$name;format="lower,word"$
```
Comment (gmethvin) [inline src/main/g8/default.properties]: It's kind of weird that this defaults to a hyphenated value `com.example.play-scala-seed`. Should we change the name, or maybe use a different value for the package?

Comment (gmethvin) [reply src/main/g8/default.properties]: Ok nevermind. This is "lower,word" which removes non-word chars. So it would be `com.example.playscalaseed`.

**Comment by dwijnand** [comment]
This PR is very old.  Do we want to merge it?

**Comment by octonato** [comment]
I'm closing this PR because of lack of activity.

**Pull request closed without merging**